src/gui/editorwidgets/qgsrelationwidgetwrapper.cpp

```diff
@@ -71,6 +73,11 @@ void QgsRelationWidgetWrapper::setVisible( bool visible )
     mWidget->setVisible( visible );
 }
 
+void QgsRelationWidgetWrapper::onRelatedFeaturesChanged()
+{
+  emit relatedFeaturesChanged();
```

Comment:
Did you know you can directly connect signals?

```
connect( relationEditorWidget, &QgsAbstractRelationEditorWidget::relatedFeaturesChanged, this, &QgsRelationWidgetWrapper::relatedFeaturesChanged );
```

Reply:
Not sure what do you mean? Like the autoconnect for slots starting with "on_" keyword? I thought in QGIS we prefer to connect signal/slots explicitly

Reply:
Short example:

Basically you can avoid the slot just to re-emit a signal.

```
class A
{
  signals:
     void mySignal();
}

class B
{
  signals:
     void mySignal();
}

connect( a, &A::mySignal, b, &B::mySignal );
```

Reply:
Ah nice wasn't aware of this thanks!